src/mon/MonCommands.h

```diff
@@ -313,10 +313,10 @@ COMMAND_WITH_FLAG("mds getmap " \
 COMMAND("mds metadata name=who,type=CephString,req=false",
 	"fetch metadata for mds <who>",
 	"mds", "r", "cli,rest")
-COMMAND("mds tell " \
+COMMAND_WITH_FLAG("mds tell " \
 	"name=who,type=CephString " \
 	"name=args,type=CephString,n=N", \
-	"send command to particular mds", "mds", "rw", "cli,rest")
+	"send command to particular mds", "mds", "rw", "cli,rest", FLAG(DEPRECATED))
```

Comment:
I think we want `FLAG(OBSOLETE)`. Then we can remove the handler completely from MDSMonitor.cc as this code will catch the obsolete command:

https://github.com/jcsp/ceph/blob/7d7cc63e2a131cb9eccde9b78be9253648d01796/src/mon/Monitor.cc#L2836-L2843